server/build_event_protocol/target_tracker/target_tracker.go

```diff
@@ -467,14 +467,14 @@ func isTestCommand(command string) bool {
 	return command == "test" || command == "coverage"
 }
 
-func readRepoTargetsWithTx(ctx context.Context, env environment.Env, repoURL string, tx interfaces.DB) ([]*tables.Target, error) {
+func readRepoTargetsWithTx(ctx context.Context, env environment.Env, repoURL string, h interfaces.DB) ([]*tables.Target, error) {
```

Comment:
Can we move the logic of this function into readRepoTargets instead?

(Since it's only called from one place (I think?) and it's not using a transaction any longer, WithTx is misleading)

Reply:
Done!